addon/components/o-s-s/skeleton.ts

```diff
@@ -0,0 +1,83 @@
+import Component from '@glimmer/component';
+import { assert } from '@ember/debug';
+
+interface Row {
+  style?: string;
+}
+
+interface OSSSkeletonArgs {
+  width: number;
+  height: number;
+  multiple?: number;
+  type?: 'row' | 'column';
+  gap?: number;
+}
+
+export default class OSSSkeleton extends Component<OSSSkeletonArgs> {
+  constructor(owner: unknown, args: OSSSkeletonArgs) {
+    super(owner, args);
+
+    if (this.args.type) {
+      assert(
+        `[component][OSS::Skeleton] The @type argument should be a value of ${['row', 'column']}`,
+        ['row', 'column'].includes(this.args.type)
+      );
+    }
```

Comment:
maybe assert width & height as well as they are not optional values ?

Reply:
Ah, I see there are default values below, maybe tag them as optional in the OSSSkeletonArgs interface then ?

Reply:
So here `type` is also mandatory? It is not the case in the interface

Reply:
`if (this.args.type)` prevent mandatory